packages/graphql-kimera/src/mockProviders.js

```diff
@@ -53,7 +53,14 @@ const isResolverScenario = (node) => {
 const mergeScenarios = memoize(
   (baseScenario, assignedScenario, meta) => {
     if (isUndefined(assignedScenario) || isUndefined(baseScenario)) {
-      return assignedScenario || baseScenario;
+      if (
+        assignedScenario ||
+        assignedScenario === 0 ||
+        typeof assignedScenario === 'boolean'
+      ) {
+        return assignedScenario;
+      }
+      return baseScenario;
```

Comment:
What if `assingedScenario` is the empty string?

Since we're in an `if` branch where we know that one of the scenarios are undefined, wouldn't checking that be the future proof way of doing this? ie. 
```
if (!isUndefined(assignedScenario)) {
   return assignedScenario;
}
return baseScenario;
```